### DISCUSSION TITLE
Help with visual bottom gap (empty space?)

### DISCUSSION THREAD

**Discussion by delattre1** [LunarVim/LunarVim#2159] [Q&A] [answered]
Hi,

I started to use lvim today, (I'm using the default config) and  something is bothering me, differently from nvim, at the bottom there's an empty space (an extra line?)

Here's an image (at right is lvim, and at left neovim) to represent this difference, If someone knows what's the cause, I'll appreciate it!!

![image](https://user-images.githubusercontent.com/55804946/148704988-778decd1-c004-4ca7-ab5f-3db1d2ba383b.png)

**Comment by delattre1** [comment, marked as answer]
Setting cmdheight as 1 solved this problem ;D

**Reply by awezmehtab** [reply]
It still isn't doing anything in my case
